lib/views/reviews-view.js

```diff
@@ -270,6 +270,13 @@ export default class ReviewsView extends React.Component {
           />
           <a className="github-ReviewSummary-username" href={`https://github.com/${reviewAuthor}`}>{reviewAuthor}</a>
           <span className="github-ReviewSummary-type">{copy}</span>
+          {review.lastEditedAt !== null && (
+            <span className="github-ReviewSummary-edited">
+              &nbsp;•&nbsp;
+              <a className="github-ReviewSummary-edited" href={review.url}>edited</a>
```

Comment:
I duplicated the `github-ReviewSummary-edited` class to get the subtle color styling to apply to the link. There's probably a better way... @annthurium perhaps you can 🎨?

Same for `github-Review-edited` below...

Reply:
adding the same classname to the `span` and the `anchor` is totally fine. 

I did extract a function for rendering the link, which allowed me to only change the css in one place.